src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/AEMExtensionTests.cs

```diff
@@ -24,24 +24,48 @@ public AEMExtensionTests(Xunit.Abstractions.ITestOutputHelper output)
         }
 
         [Fact]
```

Comment:
@MSSedusch Is there a reason why these tests are not marked as check-in tests (Category.AcceptanceType, Category.CheckIn)?